src/config.rs

```diff
@@ -0,0 +1,62 @@
+use serde::Deserialize;
+use std::fs::read_to_string;
```

Comment:
style nit: for types, like `use serde::Deserialize;`, importing the type fully makes sense. However, for functions, the convention is to use the module, so

    use std::fs;

here, and then below, use `fs::read_to_string`.

Reply:
will do!